.appveyor.yml

```diff
@@ -7,7 +7,6 @@ environment:
   matrix:
     - PYTHON: "C:\\Python37-x64"
     - PYTHON: "C:\\Python36-x64"
-    - PYTHON: "C:\\Python35-x64"
```

Comment:
I support this move, but why was it done?

Reply:
It was not my plan. But it seems AppVeyor was using a very old `pip` which would not allow to interpret `black;python_version>"3.5"`. So pipelines were failing.

Tried to update `pip` from within the `.appveyor.yml` and from within `tox.ini` without success, so I decided to drop it. Feel free to try to fix it if you want to spend your time with that.

We still have Python 3.5 coverage in Travis and 3.5 will be dropped anyways next year so I did not feel too bad while dropping it... :joy: Also, AppVeyor pipelines are the slowest, and this reduces the overall CI run time.

Reply:
Yeah, sure, I don't think this is needed either. It's nice to have, but I don't think it's worth it to invest too much time into 3.5